src/actions.ml

```diff
@@ -507,7 +507,8 @@ type ci_minimization_job_suggestion_info =
   ; head_job_succeeded: bool
   ; missing_error: bool
   ; non_v_file: string option
-  ; job_kind: string (*; overlayed: bool*) }
+  ; job_kind: string
+  ; job_name: string (*; overlayed: bool*) }
```

Comment:
`job_name` might wrongly suggest that this refers to the name of the job in GitLab CI. If what is stored in there is the Makefile target, what about calling it `job_target`?

Reply:
Done